custom_components/tuya_local/devices/avatto_trv06_radiatorvalve.yaml

```diff
@@ -52,3 +53,53 @@ secondary_entities:
         type: boolean
         optional: true
         name: lock
+  - entity: binary_sensor
+    class: battery
+    category: diagnostic
+    dps:
+      - id: 35
+        type: bitfield
+        name: sensor
+        class: measurement
+        mapping:
+          - dps_val: 0
+            value: false
+          - value: true
+  - entity: switch
+    translation_key: anti_frost
+    category: config
+    dps:
+      - id: 36
+        type: boolean
+        name: switch
+        optional: true
+        mapping:
+          - dps_val: null
+            value: false
+            hidden: true
+  - entity: switch
+    name: Antiscale
+    category: config
+    icon: "mdi:shimmer"
+    dps:
+      - id: 39
+        type: boolean
+        name: switch
+        optional: true
+        mapping:
+          - dps_val: null
+            value: false
+            hidden: true
+  - entity: number
+    name: Temperature calibration
+    category: config
+    icon: "mdi:thermometer-check"
+    dps:
+      - id: 47
+        type: integer
+        optional: true
+        name: value
+        unit: C
```

Comment:
Unit needs to be ° since we need to avoid unit conversion in the case the user has their display preference in HA set to F, so that 0 is always 0.